definitions/variable/emissions/emissions.yaml

```diff
@@ -251,18 +266,17 @@
     description: Emissions of {Level-2 Species} from waste (IPCC category 6)
     unit: "{Level-2 Species}"
 
-- Emissions|{Level-2 Species}|Fossil Fuel Fires:
-    description: Emissions of {Level-2 Species} from fossil fuel fires (IPCC category 7)
-    unit: "{Level-2 Species}"
-
 - Emissions|{Level-2 Species}|Other:
-    description: Emissions of {Level-2 Species} from other sources
+    description: Emissions of {Level-2 Species} from other sources, (IPCC category 7, including fossil fuel fires)
     unit: "{Level-2 Species}"
```

Comment:
This variable is a little ambiguous. In any processing, I would just assume that it is a flux from some fossil reservoir to the atmosphere but that could be wrong, depending on what it actually is.

Reply:
In part that is because the IPCC category also is "Only use this category exceptionally, for any categories than cannot be accommodated in the categories described above. Include a reference to where a detailed explanation of the category can be found."

Maybe I'll just add that.

Reply:
How's this d4fad19b34d41528f5db215191d91a999f9566ea